File: Iniciante/1008 - Salário/Python/1008.py
def main():
    employee_number = int(input())
    hours_worked = int(input())
    hourly_rate = float(input())
    wage = hours_worked * hourly_rate
    print(f"NUMBER = {employee_number}\nSALARY = U$ {wage:.2f}")


if __name__ == "__main__":
    main()

File: Iniciante/1008 - Salário/TypeScript/1008.ts
import * as fs from "fs";

function main(): void {
  const input: string = fs.readFileSync("/dev/stdin", "utf8");
  const lines: string[] = input.split("\n");
  const [employeeNumber, hoursWorked, hourlyRate] = lines.map(Number);
  const wage: number = hoursWorked * hourlyRate;
  console.log(`NUMBER = ${employeeNumber}\nSALARY = U$ ${wage.toFixed(2)}`);
}

main();
